tsconfig.json

```diff
@@ -6,7 +6,7 @@
     "esModuleInterop": true,
     "forceConsistentCasingInFileNames": true,
     "lib": ["ES2019", "DOM"],
-    "module": "commonjs",
+    "module": "esnext",
```

Comment:
does it works with `es2020` as well? maybe we can use it to be futurstic by using modules but be a bit preservative at the same time by using a well supported standard?

Reply:
Done!